specification/imds/data-plane/readme.md

```diff
@@ -21,7 +21,7 @@ These are the global settings for the Instance Metadata Service API.
 
 ``` yaml
 openapi-type: data-plane
-azure-arm: true
+azure-arm: false
```

Comment:
Can you remove the `csharp` block starting on line 295 of this document.

Reply:
Looks like this change to line 24 was lost, and the `output-folder: ./Generated` on line 26 still needs to be removed.